sympy/utilities/mathml/data/omtocmml.xsl

```diff
@@ -0,0 +1,1397 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- ********************************************************** -->
+<!-- XSL Transform of OpenMath to Content MathML                -->
+<!-- (Based on initial version by David Carlisle)               -->
+<!--                                                            -->
+<!-- Author: Clare M. So <clare@scl.csd.uwo.ca>                 -->
```

Comment:
I know this is a old thread. Sorry, I have not found it until now.  This is me. :-)

I can check with my former boss if it is ok to use this stylesheet in any future work.